src/base/bittorrent/sessionimpl.h

```diff
@@ -749,6 +764,9 @@ namespace BitTorrent
         bool m_IPFilteringConfigured = false;
         mutable bool m_listenInterfaceConfigured = false;
 
+        QString m_additionalTrackersFromURL;
```

Comment:
Seems you don't really need this since you already have `m_additionalTrackerEntriesFromURL`.

Reply:
This is mostly useful for the comparison against the existing tracker list to determine if we need to call `populateAdditionalTrackersFromURL()`, though it's also used to get the list of trackers to display. We could instead do the comparison against `QList<TrackerEntry> m_additionalTrackerEntriesFromURL ` and parse this every time, but it seemed easier to keep around the `QString`.

Reply:
OK. Lets keep it as is for now.